packages/react-dom/unstable-native-dependencies.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
```

Comment:
Since there's a new file we need to add it to whitelist in package.json, or it would get ignored on publish.